managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3663,7 +3663,7 @@ Pair<PositionImpl, Long> getFirstPositionAndCounter() {
 
     public void activateCursor(ManagedCursor cursor) {
         synchronized (activeCursors) {
-            if (activeCursors.get(cursor.getName()) == null) {
+            if (!cursor.isActive()) {
```

Comment:
If any refactoring were to be made, I think we should have switched to use the `ManagedLedgerImpl#isCursorActive` method.